### PR TITLE
Update Microsoft.IdentityModel.Protocol.Extensions version to 1.0.3.42

### PR DESCRIPTION
**Pull request by ChenYan98**
Issue #2

### CODE REVIEW

File: TaskWebApp/TaskWebApp.csproj [hunk 1]
```diff
@@ -84,9 +84,9 @@
     <Reference Include="System.Data.Entity" />
     <Reference Include="System.Drawing" />
     <Reference Include="System.IdentityModel" />
-    <Reference Include="System.IdentityModel.Tokens.Jwt, Version=4.0.20622.1351, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\System.IdentityModel.Tokens.Jwt.4.0.2.206221351\lib\net45\System.IdentityModel.Tokens.Jwt.dll</HintPath>
+    <Reference Include="System.IdentityModel.Tokens.Jwt, Version=4.0.30826.1200, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\System.IdentityModel.Tokens.Jwt.4.0.3.308261200\lib\net45\System.IdentityModel.Tokens.Jwt.dll</HintPath>
+      <Private>True</Private>
```
Comment (polita) [inline TaskWebApp/TaskWebApp.csproj]: Remove Private and add back in SpecificVersion (unless we have a reason not to go against the specific version).

File: TaskWebApp/TaskWebApp.csproj [hunk 2]
```diff
@@ -48,9 +48,9 @@
       <HintPath>..\packages\Antlr.3.5.0.2\lib\Antlr3.Runtime.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Microsoft.IdentityModel.Protocol.Extensions, Version=1.0.2.33, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Microsoft.IdentityModel.Protocol.Extensions.1.0.2.206221351\lib\net45\Microsoft.IdentityModel.Protocol.Extensions.dll</HintPath>
+    <Reference Include="Microsoft.IdentityModel.Protocol.Extensions, Version=1.0.3.42, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.IdentityModel.Protocol.Extensions.1.0.3.308261200\lib\net45\Microsoft.IdentityModel.Protocol.Extensions.dll</HintPath>
+      <Private>True</Private>
```
Comment (polita) [inline TaskWebApp/TaskWebApp.csproj]: Remove Private and add back in SpecificVersion (unless we have a reason not to go against the specific version).

**Comment by polita** [comment]
The only change is now to update the version numbers of the dependent assemblies. Since the assemblies are compatible, this isn't an important change and actually limits which versions customers can use to run the sample against. Do we need to make this change?

**Comment by derisen** [comment]
Closing as this repo is being archived.

**Pull request closed without merging**